joomla/joomla-cms
# [5.0] Archive content schemaorg

Pull Request for Issue # .

### Summary of Changes
Over agressive search and replace when the itemprop values were removed broke all the link on an archived article.


### Testing Instructions
Create an archive article and a menu link for archive articles
Check the article in the front end

### Actual result BEFORE applying this Pull Request
 All the links are all fubar due to missing "


### Expected result AFTER applying this Pull Request
All the links are now correct


### Link to documentations
Please select:
